### PR TITLE
adjusted code to numberOfEvents tests

### PR DESCRIPTION
**Pull request by decisiveHoneyBadger**

### CODE REVIEW

File: src/NumberOfEvents.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class NumberOfEvents extends Component {
         errorText: '',
       });
     }
-    this.props.updateNumberOfEvents(event.target.value);
+    this.props.updateEvents(event.target.value);
```
Comment (stratospherique) [inline src/NumberOfEvents.js:20]: You need to add a check here to verify that the prop `updateEvents` was passed.
```JavaScript
if (this.props.updateEvents) {
  this.props.updateEvents(event.target.value);
}
```

File: src/_tests_/App.test.js [hunk 2]
```diff
@@ -74,17 +74,6 @@ describe('<App /> integration', () => {
...
-    );
-    const eventObject = { target: { value: '15' } };
-    numberInput.at(0).simulate('change', eventObject);
-    expect(AppWrapper.state('numberOfEvents')).toBe('15');
-    AppWrapper.unmount();
-  });
-
   test('changing the number of events', () => {
     const AppWrapper = mount(<App />);
     const EventListWrapper = AppWrapper.find('.EventList');
```
Comment (stratospherique) [inline src/_tests_/App.test.js:79]: The title of the tests implies that the number of events input is going to change, yet the logic inside the tests doesn't change the value of the input. So either add an action where the input gets changed, or change the test title to something like: **returns the maximum of available events when no number gets introduced in the number of events input**

Comment (stratospherique) [reply src/_tests_/App.test.js:79]: Also, `expect(EventListWrapper.find('.EventList')).toHaveLength(1);` is something obvious that doesn't need to be tested within integration tests. You can test however the number of child event elements of the .EventList container.
 `expect(EventListWrapper.find(Event)).toHaveLength(mockData.length);`
